lib/src/linting/handle-lint-violations.ts

```diff
@@ -0,0 +1,69 @@
+import { GroupedLintRuleViolations } from "./rule";
+import { SpotConfig } from "cli/src/commands/lint";
+
+/**
```

Comment:
This JSDoc looks like it was meant to be on `handleLintViolations()`, not on the `HandleLintViolationsDependencies` interface?